blocks/search/search.css

```diff
@@ -1,163 +1,167 @@
-.search__box {
-  display: flex;
-  position: relative;
-  justify-content: flex-end;
-}
-
-.search__input-wrapper {
-  transition: all 130ms ease;
-  display: flex;
-  flex-direction: row;
-  width: 0;
-  opacity: 0;
-  right: 2rem;
-}
+.search {
+  --search-input-wrapper-width: 16.5rem;
+  --search-input-margin-block-end: 1rem;
+
+  .search__box {
+    display: flex;
+    position: relative;
+    justify-content: flex-end;
+  }
 
-.search__box--expanded .search__input-wrapper {
-  width: 100%;
-  opacity: 1;
-
-  &::before {
-    content: "";
-    width: 1.25rem;
-    height: 100%;
-    background-size: contain;
-    background: var(--icon-magnify) center no-repeat;
-    position: absolute;
-    inset-inline-start: 0.5rem;
+  .search__input-wrapper {
+    transition: all 130ms ease;
+    display: flex;
+    flex-direction: row;
+    width: 0;
+    opacity: 0;
   }
 
-  @media (min-width: 48rem) {
-    width: 16.5rem;
+  .search__box--expanded .search__input-wrapper {
+    width: 100%;
+    opacity: 1;
+
+    &::before {
+      content: "";
+      width: 1.25rem;
+      height: 100%;
+      background: var(--icon-magnify) center no-repeat;
+      background-size: contain;
+      position: absolute;
+      inset-inline-end: var(--search-input-wrapper-width);
```

Comment:
changed this from a lil padding to be from the opposite side, which resolves that search-as-a-block bug i'm not apologizing for

Reply:
also i like this better anyway